.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -8,57 +8,68 @@ body:
+          required: true
+        - label: I already searched in Google "How to X in LangChain" and didn't find any information.
+          required: true
+  - type: textarea
+    id: reproduction
     validations:
       required: true
+    attributes:
+      label: Example Code
+      description: |
+        Please add a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) with your use case.
+        
+        If a maintainer can copy it, run it, and see it right away, there's a much higher chance that you'll be able to get help.
+        
+        If you're including an error message, please include the full stack trace not just the last error.
+        
+        Important! Use code tags to correctly format your code. See https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting
+        Avoid screenshots when possible, as they are hard to read and (more importantly) don't allow others to copy-and-paste your code.
 
+      placeholder: |
+        The following code: 
+        
+        ```python
+        
+        from langchain import ....
+        # Some code goes here
+        ```
+        
+        Error / Full Stack trace:
   - type: textarea
-    id: who-can-help
+    id: description
     attributes:
-      label: Who can help?
+      label: Description
       description: |
-        Your issue will be replied to more quickly if you can figure out the right person to tag with @
-        If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
-
-        The core maintainers strive to read all issues, but tagging them will help them prioritize.
-
-        Please tag fewer than 3 people.
-
-        @hwchase17 - project lead
-
-        Tracing / Callbacks
-        - @agola11
-
-        Async
-        - @agola11
-
-        DataLoader Abstractions
-        - @eyurtsev
+        What is the problem, question, or error?
 
-        LLM/Chat Wrappers
-        - @hwchase17
-        - @agola11
-
-        Tools / Toolkits
-        - ...
-
-      placeholder: "@Username ..."
-
-  - type: checkboxes
-    id: information-scripts-examples
+        Write a short description telling what you are doing, what you expect to happen, and what is currently happening.
+      placeholder: |
+        * I'm trying to use the `langchain` library to do X.
+        * I expect to see Y.
+        * Instead, it does Z.
+    validations:
+      required: true
+  - type: textarea
+    id: system-info
     attributes:
-      label: Information
-      description: "The problem arises when using:"
-      options:
-        - label: "The official example notebooks/scripts"
-        - label: "My own modified scripts"
-
+      label: System Info
+      description: Please share your system info with us.
+      placeholder: LangChain version, platform, python version, ...
```

```suggestion
      placeholder: `pip freeze | grep langchain`, platform, python version
```
Will want versions of other packages they're using (core/partner/etc)